content/How to create an Activity Indicator with SwiftUI - Pablo Malvasio.md

```diff
@@ -0,0 +1,210 @@
+![Main image](images/spinner-cover.jpg)
+
+# How to create an Activity Indicator with SwiftUI
+
+## Introduction
+
+The year was 2019 (far far away from any COVID-19 or natural disasters) and me as many other developers were at the San Jose McEnery Convention Center waiting for all the new announcements Craig, Tim and all the engineers from Apple were about to share with us.  
+
+Thats's when SwiftUI took us by storm. We’re delighted by this new way to create our UI on a declarative way and let the data flow between the views with all the new tools SwiftUI and Combine provide. This new addition to the Apple family was huge, and even that was too early for production, we couldn’t stop ourselves to start playing around with it. But this "beta" state started to being noticed when some performance issues or even lack of support for some primitive components that were part of our daily work appeared. 
+
+One of them is the lack of a native Activity Indicator. Lucky us, we can host UIKit views on our SwiftUI code, but we know that’s tempting to create our own Spinner made 100% with this new tools.
+
+
+## Overview
+
+In this tutorial, we’ll be building a simple but functional SpinnerView from scratch using entirely SwiftUI views. 
+
+We’ll animate and define a quick way to call the spinner when we need it, and who knows, maybe we have time to make it customizable.
+
+Down bellow you can follow the steps to create the Spinner, but if you’re to lazy to follow all the steps, [here](https://github.com/rootstrap/spinner-playground) you can find a small project with the final version to copy & paste into your apps.
+
+## Lets do this
+
+### The Basics
+
+For this tutorial I’ll assume you already have some knowledge about SwiftUI and how the view behaves, or what is a modifier. 
+
+So, to start if you don’t a have a project to add the spinner, you can create one with SwiftUI as base framework or you can just go to your existing project and create a new SwiftUI file to start working.
+
+In this case we’ll be creating a rounded square with a spinning view in the middle, a full blurred background and some shadows just in order to say that we made something fancy. The final result will look like this: 
```

Comment:
how about: `a full blurred background and some shadows **for extra fanciness**`

Reply:
Me like it 🤣

Reply:
💅